lib/kitchen/note_element.rb

```diff
@@ -62,10 +58,26 @@ def self.is_the_element_class_for?(node)
 
     protected
 
-    def translation_key_in(possible_translation_keys)
-      keys = possible_translation_keys & classes
+    def key
```

Comment:
`key` is maybe too generic for this method name.  `detected_note_title_key` or something is probably better